pre-stop.d/10_remove_legacy_microsoft_plugin.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+set -euo pipefail
+
+sentinel="/var/lib/xivo-upgrade/10_remove_legacy_microsoft_plugin"
+
+[ -e "$sentinel" ] && exit 0
+
+# Remove the obsolete package built by wazo-plugind that will conflict
+# the new one
+pkg_name="wazo-plugind-wazo-microsoft-official"
+output=$(dpkg-query -W -f '${Status}' "$pkg_name" 2>/dev/null || true)
+if [ "$output" == "install ok installed" ]; then
+    apt-get purge -y "$pkg_name"
```

Comment:
You don't need that part ?

```
    # NOTE(sileht): postrm uses the rules file that no longer on the server
    # during purge, so delete it.
    rm -f /var/lib/dpkg/info/wazo-plugind-wazo-push-mobile-official.postrm
```

Reply:
No because the postrm contains `|| true` when restarting services
https://github.com/TinxHQ/wazo-google/blob/master/wazo/rules#L22-L23

Reply:
Even with this, the error will occurs, the postrm generated by plugind looks buggy: https://paste.debian.net/1091837/

Reply:
thats bad, I opened a ticket to investigate/fix plugind

Reply:
I pushed a new version removing the postrm script